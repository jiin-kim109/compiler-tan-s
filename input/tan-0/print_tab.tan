main {
	const a := 5;
	print \ta \n;
	print "test"\t a\n;
}