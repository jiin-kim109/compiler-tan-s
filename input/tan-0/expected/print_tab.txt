        5
test    5
